transfer-items.lic

```diff
@@ -0,0 +1,48 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#transfer-items
+=end
+
+custom_require.call(%w[common])
+
+class ItemTransfer
+  include DRC
+
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'source', regex: /^[A-z\s\-]+$/i, variable: true, description: 'Source container' },
+        { name: 'destination', regex: /^[A-z\s\-]+$/i, variable: true, description: 'Destination container' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
+    @source = args.source
+    @destination = args.destination
+
+    transfer_items
+  end
+
+  def transfer_items
+    fput("look in my #{@source}")
+
+    while line = get
+      if line =~ /In the .*#{@source} you see/
+        items = line
+        break
+      end
+    end
+
+    items = items.sub(' and ', ',')
```

Comment:
What do you think about something like this? (in place of lines 34-44)

```
    items
      .sub(' and ', ',')
      .split(',')
      .map { |full_name| full_name.split(' ') }
      .map { |parts| parts.last.sub('.', '') }
      .each do |item|
        fput("get #{item} from my #{@source}")
        fput("put #{item} in my #{@destination}")
      end
```

Reply:
that works great actually, thank you for the nicer way to do it